LX/src/main/java/heronarts/lx/LXSnapshotValue.java

```diff
@@ -0,0 +1,64 @@
+package heronarts.lx;
+
+import com.google.gson.JsonObject;
+
+import heronarts.lx.parameter.LXParameter;
+
+public class LXSnapshotValue extends LXComponent {
+
+  public Boolean isEnabled;
+
+  public double value;
+
+  public LXParameter parameter;
+
+  public LXSnapshotValue(LX lx) {
+    super(lx);
+  }
+
+  public LXSnapshotValue(LX lx, LXComponent scope, JsonObject obj) {
+    super(lx);
+    this.isEnabled = obj.get(KEY_ENABLED).getAsBoolean();
+    this.value = obj.get(KEY_VALUE).getAsDouble();
+    this.parameter = getParameter(lx, scope, obj.getAsJsonObject(KEY_PARAMETER));
+  }
+
+  public static LXParameter getParameter(LX lx, LXComponent scope, JsonObject obj) {
```

Comment:
Looks redundant with LXParameterModulation.getParameter()... this should probably be pushed up to being a more generic static method on the LXComponent class...

Reply:
Yes.